bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java

```diff
@@ -41,21 +39,51 @@
  * <ul>
  * <li>All API languages supported</li>
  * <li>Only default voice supported with good audio quality</li>
- * <li>Only MP3, OGG and AAC audio formats supported</li>
+ * <li>MP3, OGG, AAC and WAV audio formats supported</li>
  * <li>It uses HTTP and not HTTPS (for performance reasons)</li>
  * </ul>
  *
  * @author Jochen Hiller - Initial contribution
  * @author Laurent Garnier - add support for all API languages
  * @author Laurent Garnier - add support for OGG and AAC audio formats
+ * @author Andreas Brenk - add support for WAV audio format
  */
 public class VoiceRSSCloudImpl implements VoiceRSSCloudAPI {
 
     public static final String DEFAULT_VOICE = "default";
 
     private final Logger logger = LoggerFactory.getLogger(VoiceRSSCloudImpl.class);
 
-    private static final Set<String> SUPPORTED_AUDIO_FORMATS = Stream.of("MP3", "OGG", "AAC").collect(toSet());
+    private static final Set<AudioFormat> SUPPORTED_AUDIO_FORMATS = Set.of(
+            new AudioFormat(AudioFormat.CONTAINER_NONE, AudioFormat.CODEC_MP3, null, 16, null, 44_100L),
```

Comment:
Bitrate was previously set to 64000 for MP3. isn't it any problem to set it to null now ?

Reply:
It was specified, yes, but as it was not sent to the VoiceRSS API it was not guaranteed to be correct. And AudioFormat's interpretation of "null" is "any value", so I think it is better this way.

I don't know if the bitRate field in AudioFormat is used anywhere at all, really. It's not used in openhab-addons and only copied around in openhab-core...

Reply:
If not used, of course it is ok.